dependency-graph/dependency-review.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+jobs:
```

Comment:
Please add the [permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token) which this job would need.

Reply:
My understanding as per [this](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#how-the-permissions-are-calculated-for-a-workflow-job) and discussions with the team is, our `GITHUB_TOKEN` for post-GA installs will only require `repo: read` permissions and might be fine with the `GITHUB_TOKEN` defaults?

cc @febuiles @hmaurer to confirm ?

Reply:
It would be good practice to nonetheless put "repo: read" permission as this explicitly declares that this workflow only needs such permission. Even if org/repo admin change the default GITHUB_TOKEN to be more permissive or restrictive, the workflow will have just the right level of security.

Reply:
OK will do! thank you!

Reply:
done 👍 thank you!